src/checkstyle/suppressions.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<!DOCTYPE suppressions PUBLIC
+  "-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
+  "https://checkstyle.org/dtds/suppressions_1_2.dtd">
+<suppressions>
+  <suppress files="com[\\/]google[\\/]cloud[\\/]graalvm[\\/]features[\\/]substitutions[\\/]" checks="TopLevelClass|TypeName" />
```

Comment:
You probably don't need this anymore after changing the class name.

Reply:
Ah good catch - removed.